addons/hr_timesheet/models/project.py

```diff
@@ -45,10 +45,24 @@ class Project(models.Model):
         help="Total number of time (in the proper UoM) recorded in the project, rounded to the unit.")
     encode_uom_in_days = fields.Boolean(compute='_compute_encode_uom_in_days')
     is_internal_project = fields.Boolean(compute='_compute_is_internal_project', search='_search_is_internal_project')
-    remaining_hours = fields.Float(compute='_compute_remaining_hours', string='Remaining Invoiced Time', compute_sudo=True)
+    remaining_hours = fields.Float(compute='_compute_remaining_hours', store=True, string='Remaining Invoiced Time', compute_sudo=True)
```

Comment:
I get it, you store the field to use it in `_search_remaining_hours_percentage`.
But I'm not sure what is best: storing it or than just doing the equivalent of the _read_group of `_compute_remaining_hours` in your sql request.
I guess I'll leave it to Xavier's review :man_shrugging: